aibolit/patterns/assign_null_finder/assign_null_finder.py

```diff
@@ -0,0 +1,37 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2020 Aibolit
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import re
+
+
+class AssignNullFinder:
+
+    def value(self, filename: str):
+        '''main function'''
```

Comment:
@lukyanoffpashok Please try to implement this method by using `javalang.tree` API.
Hint: `assignments = AST(filename).value().filter(Assignment)`
then you can find `null`  by traversing and looking for `Literal` with `null `value

Reply:
I thought about it, but in my opinion implementation with 're' is better than implementation with 'javalang'. In case of 'javalang' there are troubles with definition of strings' numbers because Assignment node has no attribute position.number.